packages/journalforing/src/components/journalpost/JournalpostDetaljer.tsx

```diff
@@ -116,8 +117,11 @@ const JournalpostDetaljer: FunctionComponent<OwnProps> = ({
   reserverOppgave,
   navAnsatt,
   oppdaterValgtOppgave,
+  flyttTilGosys,
 }) => {
   const skalKunneEndreSøker = !journalpost.bruker;
+  const erLokalOppgave: boolean = oppgave.kilde === OppgaveKilde.LOKAL;
```

Comment:
Trenger å deklarere med : boolean her? Mente ts forstod at dette var en boolsk uten å måtte deklarere det.

Reply:
Antageligvis ikke. Men syns det er greit å bygge på typesikkerhet ellers blir Any default??
Jeg ble bare litt inspirert for mye av kotlin og autocomplete i webstorm her regner jeg med.